slack-lint-checks/lint-baseline.xml

```diff
@@ -540,6 +540,116 @@
             column="38"/>
     </issue>
 
+    <issue
+        id="LintImplTrimIndent"
+        message="No need to call `.trimIndent()` in issue registration strings; they are already trimmed by indent by lint when displaying to users"
+        errorLine1="              .trimIndent(),"
+        errorLine2="               ~~~~~~~~~~~~">
+        <location
+            file="src/main/java/slack/lint/compose/ComposeCompositionLocalNamingDetector.kt"
+            line="44"
+            column="16"/>
```

Comment:
This is just categorically wrong and all the error messages in tests are garbled if I remove this. Maybe the IDE does trimIndent() but the test runner doesn't, but I'm favoring redundancy than yet another venue for arguing with Lint's oft-broken testability